.gitignore

```diff
@@ -3,3 +3,4 @@
 lib/
 node_modules
 usage.txt
+.idea/
```

Comment:
I'd prefer if we avoided IDE-specific exclusions. 

It's often [recommended to do that locally](https://robots.thoughtbot.com/global-gitignore) so that we don't have to worry about everyone's preferred IDEs/editors.